modules/gloo/pkg/vsfactory/target.go

```diff
@@ -51,32 +55,32 @@ func getTarget(obj *riov1.Service, systemNamespace string) (result target, err e
 		}
 	}
 
-	if obj.Status.ComputedReplicas != nil && *obj.Status.ComputedReplicas == 0 {
-		result.ScaleIsZero = true
-		result.Port = 80
-		result.OriginalTarget = struct {
-			Name      string
-			Namespace string
-		}{Name: obj.Name, Namespace: result.Namespace}
-		result.Name = constants.AutoscalerServiceName
-		result.Namespace = systemNamespace
+	result.Hosts, err = determineHosts(obj.Status.Endpoints)
+	if err != nil {
+		logrus.Warnf("%v", err)
```

Comment:
Is it for debugging purpose? I think the error will ultimately pop up and show up in logs.

Reply:
Dropped